src/applications/check-in/pre-check-in/pages/Introduction/IntroductionDisplay.jsx

```diff
@@ -60,14 +60,11 @@ const IntroductionDisplay = props => {
     },
   ];
   const privacyStatement = (
```

Comment:
Could we just drop this in the component and shorten the slug?

Reply:
yeah good point, now that we are using children in the modal, this var makes less sense. As for shortening the key, I'm a little shy about changing things with translations still being somewhat in flux. But maybe this one is fine since it is older. @acrollet What do you think?